buildahimage/upstream/Dockerfile

```diff
@@ -29,16 +34,14 @@ RUN dnf -y install --enablerepo=updates-testing --exclude=container-selinux \
      runc \
      fuse-overlayfs \
      fuse3 \
-     containers-common;
-
-# Create a directory and clone from the Buildah
-# GitHub repository, then make and install Buildah
-# to the container.
-RUN mkdir /root/buildah; \
+     containers-common; \
+     mkdir /root/buildah; \
      git clone https://github.com/containers/buildah /root/buildah/src/github.com/containers/buildah; \
      cd /root/buildah/src/github.com/containers/buildah; \
      make;\
-     make install
+     make install;\
+     rm -rf /root/buildah/*; \
+     dnf -y remove bats git golang go-md2man make;
```

Comment:
Maybe we could add a `dnf clean all` here?

Reply:
Good thought, done.